docs/index.txt

```diff
@@ -9,62 +9,172 @@ Laravel MongoDB
 .. meta::
    :keywords: php framework, odm
 
-This package adds functionalities to the Eloquent model and Query builder for
-MongoDB, using the original Laravel API.
-*This library extends the original Laravel classes, so it uses exactly the
-same methods.*
+.. toctree::
+   :titlesonly:
+   :maxdepth: 1
+
+   /install
+   /eloquent-models
+   /query-builder
+   /user-authentication
+   /queues
+   /transactions
+   /upgrade
+
+Introduction
+------------
+
+Welcome to the documentation site for the official {+odm-long+}.
+This package extends methods in the PHP Laravel API to enable you to configure
+your Laravel app to work with MongoDB as a datastore. {+odm-short+} allows you
```

Comment:
The sentence can be slight lighter to express the same thing.
```suggestion
This package extends methods in the PHP Laravel API to work with MongoDB as a datastore in your Laravel application. {+odm-short+} allows you
```

Reply:
Thanks for the suggestion! I will incorporate it into the PR.